.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: Iceberg Bug report 🐞
+description: Problems, bugs and issues with Apache Iceberg
+labels: ["kind:bug"]
+body:
+  - type: dropdown
+    attributes:
+      label: Apache Iceberg version
+      description: >
+        What Apache Iceberg version are you using?
+      multiple: false
+      options:
+        - "0.13.1 (latest released)"
+        - "0.13.0"
+        - "0.12.1"
+        - "0.12.0"
+        - "0.11.1"
+        - "0.11.0"
+        - "main (development)"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Query engine
+      description: Which query engine (Flink, Spark, EMR, etc) are you using? What version?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Please describe the bug 🐞
+      description: >
+        Please describe the problem, what to expect, and how to reproduce.
+        Feel free to include stacktraces. You can include files by dragging and dropping them here.
```

Comment:
maybe slightly update this to: `Feel free to include stacktraces and the Iceberg catalog configuration`

Reply:
Done! 👍🏻